src/main/resources/maven_templates/artifact-reference-checker-maven-plugin.xml

```diff
@@ -0,0 +1,12 @@
+<plugin>
+    <groupId>com.exasol</groupId>
+    <artifactId>artifact-reference-checker-maven-plugin</artifactId>
+    <version>0.3.1</version>
```

Comment:
How are you going to keep the version up-to-date?

Reply:
I'am not going to update it. The pkugin does not check the version. It's just a template, that is copied if the plugin does not exist.
Users should update them using maven versions plugin